part2-장현태-sprint6/sprint-mission-6.sql

```diff
@@ -0,0 +1,215 @@
+  -- # 초급 문제
+
+-- 1. `orders` 테이블에서 모든 주문을 조회하세요.
+SELECT * FROM orders;
+
+-- 2. `orders`테이블에서 `id` 가 `423`인 주문을 조회하세요.
+SELECT *
+FROM orders
+WHERE id=423;
+
+-- 3. `orders` 테이블에서 총 주문 건수를 `total_orders`라는 이름으로 구하세요.
+SELECT COUNT(id) as total_orders
+FROM orders;
+
+-- 4. `orders` 테이블에서 최신 순으로 주문을 조회하세요. (`date`, `time` 컬럼이 분리되어 있다는 점에 주의)
+SELECT *
+FROM orders
+ORDER BY date DESC, time DESC;
+
+-- 5. `orders` 테이블에서 오프셋 기반 페이지네이션된 목록을 조회합니다. 페이지 크기가 10이고 최신순일 때, 첫 번째 페이지를 조회하세요.
+SELECT *
+FROM orders
+ORDER BY date DESC, time DESC;
+LIMIT 10;
```

Comment:
명시적으로 offset까지 사용하면 좋을 것 같습니다.
```suggestion
LIMIT 10;
OFSET 0
```